<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateSoldierView</class>
 <widget class="QWidget" name="UpdateSoldierView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>156</y>
     <width>108</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="teamComboBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>88</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="squadComboBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>54</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="roleLineEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>224</y>
     <width>108</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="soldierComboBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>122</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="rankComboBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="updateButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>80</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>32</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>129</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="backButton">
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Duty</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Update Duty</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
